napari/_qt/qt_main_window.py

```diff
@@ -109,6 +110,10 @@ def event(self, e):
                 inst.append(inst.pop(inst.index(self)))
             except ValueError:
                 pass
+        if e.type() == QEvent.ToolTip and self.qt_viewer.viewer.tooltip_text:
+            QToolTip.showText(
+                e.globalPos(), self.qt_viewer.viewer.tooltip_text
+            )
```

Comment:
I think it has been a Qt bug for a long time that using QToolTip directly does not use the stylesheet, so in order to preserve this one needs to use a custom widget that emulates a tooltip.

Reply:
I do not know about this bug. 

Here result after the last commit: 

![Zrzut ekranu z 2021-05-24 16-48-18](https://user-images.githubusercontent.com/3826210/119365299-1525d900-bcb0-11eb-9148-bf3c72d8e8c7.png)